Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/kubectl.go

```diff
@@ -144,7 +144,9 @@ var shortForms = map[string]string{
 	// Please keep this alphabetized
 	// If you add an entry here, please also take a look at pkg/kubectl/cmd/cmd.go
 	// and add an entry to valid_resources when appropriate.
+	"bc":     "buildconfigs",
 	"cs":     "componentstatuses",
+	"dc":     "deploymentconfigs",
```

Comment:
These shouldn't be here. We have our own map of abbreviations
https://github.com/openshift/origin/blob/d409b31fa672e2e77940012bca8793a80b838422/pkg/cmd/util/clientcmd/shortcut_restmapper.go#L102